.doks/Makefile

```diff
@@ -1,17 +1,36 @@
 CODACY_URL?="http://k8s.dev.codacy.org"
 RELEASE_NAME?="codacy"
 NAMESPACE?="codacy"
+CHANNEL?=codacy-incubator
 
 .PHONY: deploy_to_doks
-deploy_to_doks:
-	doctl kubernetes cluster kubeconfig save $${DOKS_CLUSTER_NAME} --set-current-context
-
+deploy_to_doks: set_cluster_context
 	$(MAKE) -C ../ update_dependencies
-
 	helm upgrade --install ${RELEASE_NAME} ../codacy/ \
-	 	-f values.yaml \
+		--recreate-pods \
+		-f values.yaml \
+		-f providers.yaml \
+		--namespace ${NAMESPACE} \
+		--set global.imagePullSecrets[0].name=docker-credentials \
+		--set global.play.cryptoSecret=$${SHARED_PLAY_CRYPTO_SECRET} \
```

Comment:
For curiosity what is the difference between the $ and $$?

Reply:
Single dollar sign -> makefile var
Double dollar sign -> shell (env) var
:-)